docs/source/en/conceptual/contribution.md

```diff
@@ -271,7 +271,7 @@ with Diffusers.
 Once an example script works, please make sure to add a comprehensive `README.md` that states how to use the example exactly. This README should include:
 - An example command on how to run the example script as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/dreambooth#running-locally-with-pytorch).
 - A link to some training results (logs, models, ...) that show what the user can expect as shown [here e.g.](https://api.wandb.ai/report/patrickvonplaten/xm6cd5q5).
-- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
+- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
```

Comment:
No need to add `e.g.`

Reply:
But the first and second items have "e.g." at their "here" part 😂.

Reply:
Let's also change those while we're at it then! 😄  

In general, I think adding Latin abbreviations like "e.g" or "i.e" is not as understandable as just writing it out in plain English. But here, we don't need to say "e.g." since earlier in this sentence we already said it was an "example"

Reply:
Done.